File: Dice-C-sharp-v4/Set.cs
namespace Puzzle;
class Set
{
    public int StopPoint { get; set; } = 100;
    public int Play()
    {
        var dice = new Dice();
        int currentPointsSum = 0;
        while (currentPointsSum <= StopPoint)
        {
            currentPointsSum += dice.Throw();
        }
        return currentPointsSum;
    }
}

File: Dice-Python-v3/set.py
import dice

class Set:
    stopPoint = 0
    pointsSum = 0
    def __init__(self, stopPoint):
        self.stopPoint = stopPoint
    def play(self):
        d = dice.Dice()
        while self.pointsSum <= self.stopPoint:
            self.pointsSum += d.throw()
        return self.pointsSum


